src/hooks/useIsClient.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useState } from 'react';
+
+export const useIsClient = () => {
+  const [isClient, setIsClient] = useState(false);
+
+  useEffect(() => {
+    setIsClient(true);
+  }, []);
+
+  return isClient;
```

Comment:
This is a hook to make sure your are on the client only.
This is to prevent hydratation error. I will try to add a comment on this.

Reply:
isBrowser is to know if we are on the browser and have access to browser API (not related to React here)
useIsClient is tied to React and hydratation, when isClient is true you can run some code on the client without worried about hydratation issue. Maybe we can find another name for it. isBrowser can't help here because on 1st render when hydratation happen your are in the browser but the client should render the same that is render in the server. Maybe useIsClientReady is better?

Reply:
Oh ok, thanks for the explanation 😄 I thought they were doing the same check, but in a different way :) 
I had no pb with the name, but yes I think `useIsClientReady()` is better :)

Reply:
I do like `useIsClientReady()`, remind me of those JQuery days.

![bitmoji](https://sdk.bitmoji.com/render/panel/0588ee45-e8ad-4364-84af-264734478d38-a1987ddb-a11e-49bc-8a9b-052f30b8f6ce-v1.png?transparent=1&palette=1&width=246)